cpp/include/cugraph_c/algorithms.h

```diff
@@ -136,45 +138,50 @@ cugraph_error_code_t cugraph_personalized_pagerank(
   cugraph_error_t** error);
 
 /**
- * @brief     Opaque bfs result type
+ * @brief     Opaque paths result type
+ *
+ * Store the output of BFS or SSSP, computing predecessors and distances
+ * from a seed.
  */
 typedef struct {
   int align_;
```

Comment:
Should we better use int32_t instead of int if the intention here is to really enforce 4 byte alignment?

Reply:
Good point.  Will fix in the implementation PR.